### ISSUE TITLE
Does not match Kotlin code style for Chained Wrapping Calls

### ISSUE THREAD

**Issue by ScottPierce** [shyiko/ktlint#171]
I have the following code:
```Kotlin
fun test() {
  listOf(1, 2, 3)
    .map {
      it.toString()
    }
    .filter {
      it.isNotBlank()
    }
    .map { "alsdfjadsf" }
    .map { "adsjfhasdf" }
}
```

This matches the style described by the Kotlin Style Guide: https://kotlinlang.org/docs/reference/coding-conventions.html#chained-call-wrapping

ktlint doesn't match the `with a single indent` part of the style guidelines for chained call wrapping.

Here is my `.editorconfig`:
```
[{*.kt,*.kts}]
indent_size=2
insert_final_newline=true
```

Here is my error when running ktlint:
```
<project dir>/features/base-android/src/main/java/KtLintTest.kt:4:1: Unexpected indentation (4) (it should be 6)
<project dir>/features/base-android/src/main/java/KtLintTest.kt:7:1: Unexpected indentation (4) (it should be 6)
<project dir>/features/base-android/src/main/java/KtLintTest.kt:10:1: Unexpected indentation (4) (it should be 6)
<project dir>/features/base-android/src/main/java/KtLintTest.kt:11:1: Unexpected indentation (4) (it should be 6)
```

According to the style guideline, an indent of 4 is correct, but it seems ktlint wants a continuation indent.

**Comment by shyiko** [comment]
Hi Scott. 
In addition to `indent_size=2` you need to set `continuation_indent_size=2` (otherwise [ktlint assumes 4](https://github.com/shyiko/ktlint#editorconfig)).

**Comment by ScottPierce** [comment]
@shyiko The kotlin style guide isn't that continuation indents are the same as normal indents, is it?

It's just in the case of chained styles that the continuation indent isn't used, right?

**Comment by ScottPierce** [comment]
@shyiko Did that make sense? Here is what the style [guidelines](https://kotlinlang.org/docs/reference/coding-conventions.html#chained-call-wrapping) say:

```
When wrapping chained calls, put the . character or the ?. operator on the next line, with a single indent
```

You are suggesting we make the `continuation indent` equal to the normal `indent`. That would violate other styles though. The style guide effectively says to NOT use a `continuation indent` when chaining method calls. This can be represented in IntelliJ settings like this:

<img width="359" alt="screen shot 2018-03-10 at 10 35 40 am" src="https://user-images.githubusercontent.com/871169/37243924-585d8986-244f-11e8-8521-ed9b701cb77a.png">

The [Android style guidelines](https://android.github.io/kotlin-guides/style.html#continuation-indent) are that continuation indents are still twice the normal indent: 
```
When line-wrapping, each line after the first (each continuation line) is indented at least +8 from the original line.
```

Do you see what I'm saying? What should I be doing in this case?

**Comment by shyiko** [comment]
Sure. It does make sense. It's going to be fixed in the next release (this is also why I haven't closed the ticket). 

Having said that, [Coding Conventions](https://kotlinlang.org/docs/reference/coding-conventions.html#chained-call-wrapping) do not mention continuation indent (at all), which means the default (in form of `indent_size=4` and `continuation_indent_size=4`) is technically "legal". As for Android Style Guide, see https://github.com/shyiko/ktlint/pull/134#issuecomment-369021113.

I'd say stay with the defaults. All this `indent != continuation_indent` business helps no one and only creates confusion. If you really want 2 space indent then https://github.com/shyiko/ktlint/issues/171#issuecomment-371970039 is the way to go.

**Comment by shyiko** [comment]
Fixed in [0.22.0](https://github.com/shyiko/ktlint/releases/tag/0.22.0).
